distributing/index.po

```diff
@@ -266,28 +265,23 @@ msgstr ""
 
 #: distributing/index.rst:128
 msgid "`Project structure`_"
-msgstr "`(en) Structure d'un projet <Project structure_>`_ ;"
+msgstr "`Project structure`_ ;"
```

Comment:
Pourquoi, ce n'était pas bien ? Contrairement à la plupart des références, ces pages ne font pas partie de la documentation officielle, donc nous n'espérons pas qu'elles soient traduites à l'avenir.

Reply:
En fait je trouvais ça bien mais j'ai (bêtement ?) suivi la recommandation du fichier [CONTRIBUTING.rst](https://github.com/python/python-docs-fr/blob/3.10/CONTRIBUTING.rst#liens-hypertextes) qui demande de ne pas traduire les descriptions des liens vers des ressources non francophones.

Reply:
Tant que ca prévient que la ressource est en anglais (et ça l'est, il y avait `(en)`) ça va, ça ne surprend pas l'utilisateur.

Reply:
Ok c'est noté.Je vais modifer ma PR pour garder les traductions sur les liens concernés.